### DISCUSSION TITLE
Could we put languages behind feature flags ?

### DISCUSSION THREAD

**Discussion by happysalada** [meilisearch/product#446] [Feedback & Feature Proposal]
if possible could we put different languages behind feature flags ?
At least some of them.
lindera downloads something off the internet, and that can be problematic for some OS (notably nixos).
would it be possible to put it behind a feature flag ? (even if it is enabled by default, as long as it can be disabled, it would be very helpful).

**Comment by ManyTheFish** [comment]
Hey @happysalada, you're right, some segmenter is downloading/building dictionaries at compile time. I added feature flags for every additional language support in the future versions of the tokenizer,

Thank you for your feedback!
